doc/man3/RAND_add.pod

```diff
@@ -18,12 +18,12 @@ RAND_keep_random_devices_open
 
  void RAND_keep_random_devices_open(int keep);
 
-Deprecated since OpenSSL 1.1.0, can be hidden entirely by defining
-B<OPENSSL_API_COMPAT> with a suitable version value, see
-L<openssl_user_macros(7)>:
+Deprecated:
```

Comment:
All functions deprecated in this PR should have an associated CHANGES.md entry saying what the preferred alternative is. The description of the functions should repeat that they are deprecated and what the alternative is.

Reply:
The functions were deprecated in 1.1.0.  It's a bit late to add a CHANGES entry for them.  I've reverted the description to what it was.